frappe_graphql/frappe_graphql/types/root.graphql

```diff
@@ -27,10 +27,16 @@ type SET_VALUE_TYPE {
     doctype: String!
     name: String!
     fieldname: String!
-    value: String!
+    value: DOCFIELD_VALUE_TYPE!
     doc: BaseDocType!
 }
 
+
+"""
+Supports ID | Boolean | Int | String | Float | JSONString
+"""
+scalar DOCFIELD_VALUE_TYPE
```

Comment:
@e-lobo We can use `Union` instead of `Scalar` for this case

Reply:
we cannot use union as input type..

Reply:
Ohh, didn't know that 😅